pkgs/tools/misc/coreutils/default.nix

```diff
@@ -99,13 +99,12 @@ stdenv.mkDerivation rec {
   separateDebugInfo = true;
 
   nativeBuildInputs = [
-    # autoreconfHook is due to patch, normally only needed for cygwin
-    autoreconfHook
     perl
     xz.bin
   ]
   ++ optionals stdenv.hostPlatform.isCygwin [
     # due to patch
+    autoreconfHook
```

Comment:
as per https://github.com/NixOS/nixpkgs/pull/311789/files/977ab5018bd37332386fd36af9227b7fa7d42eab#r1601167922

Reply:
Separate commit?  Doesn't really belong in one called "disable single-binary on native FreeBSD".